### ISSUE TITLE
Functions that return a table

### ISSUE THREAD

**Issue by ghost** [nettofarah/postgres-schema-ts#1]
Hi @nettofarah , this is a super useful tool. Do you know if it would be possible to generate types for functions that return a table? Like for example:

```sql
CREATE FUNCTION get_groups()
    RETURNS TABLE (
        group_id integer,
        group_name text
    )
    LANGUAGE sql
AS $$
    -- SELECT ...
$$;
```

I would expect it to generate:

```ts
export interface get_groups {
  group_id: number
  group_name: string
}
```

**Comment by nettofarah** [comment]
Hey, @joaopaulobdac. Thanks for the suggestion.
This is the first time I hear about functions that return tables.

Would you like to start a pull request? I'm happy to help out any way I can. Maybe start with an unit/integration test, and I could help fill in the blanks with some implementation code?

**Comment by ghost** [comment]
Yeah! I will be occupied in january because I have a couple of important exams in the upcoming weeks but after that I can start. You can use this type of function similarly to a view, with the benefit of being able to pass parameters to it.
